toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -316,36 +315,17 @@ build_rpm_in_chroot_no_install python3
 rm -vf $FINISHED_RPM_DIR/python3*debuginfo*.rpm
 chroot_and_install_rpms python3
 
-
-# openjdk needs fontconfig to build, which needs libxml
-build_rpm_in_chroot_no_install libxml2
-copy_rpm_subpackage python3-libxml2
-chroot_and_install_rpms libxml2
-chroot_and_install_rpms expat
-chroot_and_install_rpms freetype
-build_rpm_in_chroot_no_install fontconfig
-chroot_and_install_rpms fontconfig
-
-
-# Build OpenJDK and OpenJRE
-echo Java bootstrap version:
+# Download JDK rpms
+echo Download JDK rpms
 case $(uname -m) in
     x86_64)
-        echo $($LFS/usr/lib/jvm/OpenJDK-212-b04-bootstrap/bin/java -version)
-        build_rpm_in_chroot_no_install openjdk8
+        wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/x86_64/msopenjdk-11-11.0.13%2B8-LTS-4.x86_64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
     ;;
     aarch64)
-        echo $($LFS/usr/lib/jvm/OpenJDK-1.8.0.181-bootstrap/bin/java -version)
-        build_rpm_in_chroot_no_install openjdk8_aarch64
+        wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/aarch64/msopenjdk-11-11.0.13%2B8-LTS-4.aarch64.rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
     ;;
 esac
```

Comment:
Suggestion: I think we can now drop the whole `case` statement and use this instead:

```bash
HOST_ARCH="$(uname -m)"
wget -nv --timeout=30 https://packages.microsoft.com/cbl-mariner/2.0/preview/Microsoft/"$HOST_ARCH"/msopenjdk-11-11.0.13%2B8-LTS-4."$HOST_ARCH".rpm --directory-prefix=$CHROOT_RPMS_DIR_ARCH
```

What do you think?

Reply:
I have a slight preference for the case statement, since the URLs can be copied and used as-is without variable substitution.